base/applications/rapps/lang/no-NO.rc

```diff
@@ -19,7 +19,7 @@ BEGIN
     BEGIN
         MENUITEM "&Installere\tCtrl+Enter", ID_INSTALL
         MENUITEM "&Avinstallere\tCtrl+Del", ID_UNINSTALL
-        MENUITEM "&Endre", ID_MODIFY
+        MENUITEM "E&ndre", ID_MODIFY
         MENUITEM SEPARATOR
         MENUITEM "R&emove from Registry", ID_REGREMOVE
```

Comment:
The translated version of "Remove from Registry" is most likely going to start with the word "Fjern" so it would be better to keep "&Endre" and for the English string, change to ".. &from ..".

Reply:
&F wouldn't work for most languages. &E has a high chance to appear everywhere, because the word "Registry" contains that as well, and some languages do not translate that word.

Reply:
I'm not saying it should be &F for most languages but it would be natural for Norwegian because the eventual translation of "Remove ..." will be "&Fjern ...".

Reply:
ok, I can live with an extrawurst for no-NO. that doesn't hurt me in any way. I will change it back to &F then.

Reply:
No wait, that string isn't translated yet even for no-NO. the native translators can do that when they will translate it.